### PR TITLE
Episode 21

### PR DESCRIPTION
**Pull request by sockeqwe**
✅ Feed is valid:  http://castfeedvalidator.com/?url=https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/episode21/feed.rss

TODO's:

- [ ] Upload `.mp3` file to github release section.
- [ ] Add git tag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ Hosts: Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), Hannes Dor
 
 ### [Episodes](https://github.com/artem-zinnatullin/TheContext-Podcast/releases)
 
+##### Episode 20, Part 2: public final Agile
```
Comment (artem-zinnatullin) [inline README.md]: copy paste, I'll fix

File: feed.rss [hunk 2]
```diff
@@ -1310,7 +1310,7 @@
     <item>
       <guid>https://artemzin.com/static/thecontext/episodes/The.Context.episode.20.part1.mp3</guid>
       <title>Episode 20, Part 1: public final Agile</title>
-      <link>https://github.com/artem-zinnatullin/TheContext-Podcast/blob/master/show_notes/Episode_19_Part2.md</link>
+      <link>https://github.com/artem-zinnatullin/TheContext-Podcast/blob/master/show_notes/Episode_20_Part1.md</link>
```
Comment (artem-zinnatullin) [inline feed.rss]: oh snap

File: feed.rss [hunk 3]
```diff
@@ -1388,6 +1388,40 @@
 				<li><a href="https://edu.leankanban.com/users/david-anderson">David Anderson</a></li>
 				<li><a href="https://en.wikipedia.org/wiki/Kaizen">Kaizen</a></li>
 				</ul>]]></itunes:summary>
+    </item>
+	<item>
+      <guid>https://artemzin.com/static/thecontext/episodes/The.Context.episode.20.part2.mp3</guid>
+      <title>Episode 20, Part 2: public final Agile</title>
```
Comment (artem-zinnatullin) [inline feed.rss]: Wrong title

File: feed.rss [hunk 4]
```diff
@@ -1388,6 +1388,40 @@
 				<li><a href="https://edu.leankanban.com/users/david-anderson">David Anderson</a></li>
 				<li><a href="https://en.wikipedia.org/wiki/Kaizen">Kaizen</a></li>
 				</ul>]]></itunes:summary>
+    </item>
+	<item>
+      <guid>https://artemzin.com/static/thecontext/episodes/The.Context.episode.20.part2.mp3</guid>
```
Comment (artem-zinnatullin) [inline feed.rss]: Wrong link

File: feed.rss [hunk 5]
```diff
@@ -1388,6 +1388,40 @@
...
+      <title>Episode 20, Part 2: public final Agile</title>
+      <link>https://github.com/artem-zinnatullin/TheContext-Podcast/blob/master/show_notes/Episode_21.md</link>
+      <atom:link href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/97" rel="replies" type="text/html" />
```
Comment (artem-zinnatullin) [inline feed.rss]: Wrong link

File: feed.rss [hunk 6]
```diff
@@ -1388,6 +1388,40 @@
...
+      <enclosure length="60920969" type="audio/mpeg" url="https://artemzin.com/static/thecontext/episodes/The.Context.episode.21.mp3" />
+      <itunes:summary><![CDATA[<h1>Episode 21: Rx Must Die</h1>
+
+				<h3><a href="https://github.com/artem-zinnatullin/TheContext-Podcast">How to listen &amp; subscribe</a></h3>
+
+				<ul>
+				<li><a href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/99">Discussion after the episode</a></li>
+				</ul>
+
+				<p>This time we have a special episode: we have recorded a panel discussion at droidcon Berlin 2018 (June 25) with the name Rx Must Die.</p>
```
Comment (artem-zinnatullin) [inline feed.rss]: Maybe reword to say that we collaborated with Droidcon Berlin and they provided us a recording?

Comment (sockeqwe) [reply feed.rss]: hm, well , can you edit it ... I run out of good ideas how to write it down

**Comment by sockeqwe** [comment]
Thanks

Artem Zinnatullin :slowpoke: <notifications@github.com> schrieb am Mi., 1.
Aug. 2018, 10:06:
